mesheryctl/internal/cli/root/design/apply.go

```diff
@@ -36,25 +36,25 @@ import (
 )
 
 var (
-	skipSave    bool // skip saving a pattern
+	skipSave    bool // skip saving a design
 	patternFile string
 )
 
 var linkDocPatternApply = map[string]string{
 	"link":    "![pattern-apply-usage](/assets/img/mesheryctl/patternApply.png)",
-	"caption": "Usage of mesheryctl pattern apply",
+	"caption": "Usage of mesheryctl design apply",
 }
 
 var applyCmd = &cobra.Command{
 	Use:   "apply",
-	Short: "Apply pattern file",
-	Long:  `Apply pattern file will trigger deploy of the pattern file`,
+	Short: "Apply design file",
+	Long:  `Apply design will trigger deploy of the design file`,
```

Comment:
Not related but as we are here, we can make this long description more helpful

Reply:
Noted @Yashsharma1911